src/server/components/charts-engine/controllers/charts.ts

```diff
@@ -89,6 +90,14 @@ function prepareChartData(
     return {chart, type, links, template};
 }
 
+export const getHeaders = (req: Request) => {
+    const headers = {
+        ...req.headers,
+        ...Utils.pickZitadelHeaders(req),
```

Comment:
Can you proxy zitadel headers only if zitadel is enabled? Not only here

Reply:
done everywhere